.gitignore

```diff
@@ -1,2 +1,8 @@
+# dependencies
 node_modules/
+
+# IDE
 .idea
```

Comment:
```suggestion
.idea

# Vim
[._]*.s[a-v][a-z]
[._]*.sw[a-p]
[._]s[a-rt-v][a-z]
[._]ss[a-gi-z]
[._]sw[a-p]
```

let's help vim users :smile:

Reply:
Yeah, I love using vim

I just don't know if this PR will be merged, we would see